examples/vertx-opentracing/README.md

```diff
@@ -12,58 +12,64 @@ Manager is performed via publishing a message on a shared destination.
 
 The other operation that can be performed by an end user is to list the current orders associated with an account id.
 
-== Run
+## Run
 
-First step is to start up the APM server. Then open a console window from this folder and run:
+First step is to start up the APM server. Then open a console window from this folder and either run the command directly using maven:
 
-[source,shell]
-----
+```shell
 $ export HAWKULAR_APM_URI=http://localhost:8080
 $ export HAWKULAR_APM_USERNAME=...
```

Comment:
Note that we have also jdoe:password in our docs: 
https://hawkular.gitbooks.io/hawkular-apm-user-guide/content/quickstart/index.html and if they are using other user they can still change it